crates/factor-wasi/src/wasi_2023_10_18.rs

```diff
@@ -14,12 +13,7 @@ mod bindings {
 
     wasmtime::component::bindgen!({
         path: "../../wit",
-        interfaces: r#"
-            // NB: this is handling the historical behavior where Spin supported
-            // more than "just" this snaphsot of the proxy world but additionally
-            // other CLI-related interfaces.
```

Comment:
Is this comment no longer relevant?

Reply:
Yeah I think that this is an outdated comment already because the comment implies there's more than one `include` or something beyond proxy but there's just the proxy world there now.